stencil-library/src/components/payment-form/payment-form.tsx

```diff
@@ -103,7 +103,7 @@ export class PaymentForm {
               onClick={(event) => this.submit(event)}
               disabled={!this.submitButtonEnabled}
               type="submit"
-              class="btn btn-primary">
+              class="btn btn-primary submit-button">
```

Comment:
Do we need a new class or can we just override bootstrap's `btn` class?

Reply:
I'm ok with overriding it if it won't apply anywhere outside of the `payment-form.tsx` component file.

Reply:
Tried it and turns out we run into a rule specificity issue. Using a new class both solves that and keeps it better scoped.